workspaces/adventure-pack/goodies/python3/src/flatten/__init__.py

```diff
@@ -0,0 +1,9 @@
+from typing import Generator, Any
```

Comment:
In other typing systems I've seen there's usually a safe version of "I don't know what this is" and an unsafe one. For example in TypeScript `any` is the unsafe version, because the typechecker will let you do anything with an `any` whereas `unknown` is the safe one, because the typechecker won't really let you do anything with an `unknown` value until you've checked if it's an instance of some more specific type.

I'm not as familiar with Python typing, so I'm not sure whether `Any` is considered evil or not. Is there another way of saying "I don't know what this is"?

Reply:
I believe Any is Evil in python too. I think I made it better now.